lib/src/obsoleted/metrics_analysis_recorder.dart

```diff
@@ -17,14 +16,14 @@ class MetricsAnalysisRecorder
   String? _fileGroupPath;
   String? _relativeGroupPath;
   Map<ScopedClassDeclaration, Report>? _componentRecords;
-  Map<ScopedFunctionDeclaration, FunctionRecord>? _functionRecords;
+  Map<ScopedFunctionDeclaration, Report>? _functionRecords;
   List<Issue>? _issues;
   List<Issue>? _designIssues;
```

Comment:
So what do you think about storing FileReport instead of multiple fields? Or create some sort of a builder?

Reply:
I will rewrite this code block in next time